src/execute.js

```diff
@@ -219,7 +236,7 @@ export function baseUrl({spec, scheme, contextUrl = ''}) {
 // Add security values, to operations - that declare their need on them
 export function applySecurities({request, securities = {}, operation = {}, spec}) {
   const result = assign({}, request)
-  const {authorized = {}, specSecurity = {}} = securities
+  const {authorized = {}, specSecurity = []} = securities
```

Comment:
Because of variable type. When `{}` there will be exception  on `forEach` it: 'forEach is not a function'
    const {authorized = {}, specSecurity = []} = securities    
    const security = operation.security || specSecurity
  
    security.forEach